code/modules/hydroponics/seeds.dm

```diff
@@ -160,7 +160,10 @@
 	var/list/result = list()
 	var/output_loc = parent.Adjacent(user) ? user.loc : parent.loc //needed for TK
 	var/product_name
-	while(t_amount < getYield())
+	var/yield_amount = getYield()
+	if(get_gene(/datum/plant_gene/trait/hotbox))
+		yield_amount = 3 //this is now hard set to only produce 3 items
```

Comment:
Have it clamp between 0 and 3 rather than just potentially boosting a 0 into 3.

It also should update when using the harvest scanner, because it's deceptive right now.
A ten yield could suddenly become 3 without any player feedback.

There also should be a more generic method rather than simply checking for specific genes, since this could potentially be expanded to multiple genes with this effect.